src/hotspot/share/utilities/bitMap.hpp

```diff
@@ -315,6 +322,15 @@ class BitMap {
   // aligned to bitsizeof(bm_word_t).
   idx_t get_next_one_offset_aligned_right(idx_t l_index, idx_t r_index) const;
 
+  static constexpr idx_t NotFound = ~(idx_t)0;
+  // Find the index of the first bit set scanning from r_index (inclusive) to
+  // l_index (inclusive) backwards.
+  // Returns that index or NotFound if there is no such bit in the range.
```

Comment:
I think that, for consistency, it might be good for the forward search to also
return NotFound when appropriate. That's something that should be done as a
followup though, I think.

Reply:
I agree that this should not be done here - this will cause quite a few additional changes I think.